package.json

```diff
@@ -23,8 +23,10 @@
   },
   "dependencies": {
     "@guardian/src-button": "^0.15.1",
```

Comment:
Since you're using the Button component, you'll probably want to use the [FocusManager](https://www.theguardian.design/2a1e5182b/p/6691bb-accessibility/t/330e47) to disable the focus styling on button click

Reply:
This will be considered as a separate PR for Frontend and DCR. We need to discuss with the team whether we can accept the global changes produced by calling `onlyShowFocusOnTabs` across both platforms.